### PR TITLE
fix: permit simulation design

### PR DESCRIPTION
**Pull request by OGPoyraz**
## **Description**

This PR does a small alignment and pill padding update on permit simulations.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/26186?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2845

Follow-up Cherry-picked PR: https://github.com/MetaMask/metamask-extension/pull/26518

## **Manual testing steps**

1. Go to test-dapp, trigger permit signature to verify change

### **Before**

Screenshot 2024-07-29 at 15 32 11

### **After**

Screenshot 2024-07-29 at 15 31 55

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by matthewwalsh0 — DISMISSED** [review]

**Review by digiwand — DISMISSED** [review]

File: ui/components/app/confirm/info/row/row.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ export const ConfirmInfoRow: React.FC<ConfirmInfoRowProps> = ({
         display={Display.Flex}
         flexDirection={FlexDirection.Row}
         justifyContent={JustifyContent.center}
-        alignItems={AlignItems.center}
+        alignItems={AlignItems.flexStart}
```
Comment (jpuri) [inline ui/components/app/confirm/info/row/row.tsx:109]: We need to ensure that it looks correct for all other cases.

Comment (OGPoyraz) [reply ui/components/app/confirm/info/row/row.tsx:109]: Is there any special one that requires centralised label that you know?

**Review by digiwand — DISMISSED** [review]

**Review by jpuri — APPROVED** [review]

**Review by digiwand — APPROVED** [review]
Updated snapshots and verified the UI on chrome 👍🏼

**Comment by digiwand** [comment]
Follow-up cherry-pick PR into 12.1.0: https://github.com/MetaMask/metamask-extension/pull/26518

Follow-up related fix to include decimals in calculation and tooltip when ellipsis is shown on fiat: https://github.com/MetaMask/metamask-extension/pull/26523

**Pull request merged**